tests/runtest.proj

```diff
@@ -118,7 +121,7 @@ $(_XunitEpilog)
   </ItemGroup>
   <Import Project="$(SourceDir)dir.targets" />
   <PropertyGroup>
-     <OutDir>$(XunitTestBinBase)\$(Category)\</OutDir>
+     <OutDir>$(XunitTestBinBase)\$(CategoryWithSlash)\</OutDir>
```

Comment:
Might be more clear if I changed it to "CategoryPath" - I had to replace slashes with periods in "Category", since otherwise the wrappers would have been named "x/y.XunitWrapper" instead of "x.y.XunitWrapper", but I still needed to point OutDir at a path with a slash, so I created 2 properties (Has nothing to do w/ the trailing slash).